packages/playground/website/src/components/open-site-manager-button/index.tsx

```diff
@@ -0,0 +1,32 @@
+import css from './style.module.css';
+
+export function OpenSiteManagerButton() {
+	const onClick = () => {
+		// TODO use a shared function for switching views
+		const url = new URL(window.location.toString());
+		url.searchParams.set('view', 'manager');
+		window.location.assign(url.toString());
+	};
+	return (
+		<button
```

Comment:
Let's use the WordPress button component here. This one has a few issues, e.g. the click area is quite small.

Reply:
Replaced with NavigatorButton